source/process/sg_general-guidelines.rst

```diff
@@ -0,0 +1,92 @@
+.. _general-guidelines:
+
+General Guidelines
+==================
+
+The Mattermost documentation must be of high quality. It must be accurate and clear, and be presented with a style and tone that is appropriate for technical content. People who use Mattermost rely on the documentation to get their jobs done. We don't want to see an installation of Mattermost delayed because the documentation has an error or is difficult to understand.
+
+Style Guide Applicability
+-------------------------
+
+The style guide applies to all end-user, developer, and administrator documentation that appears on the Mattermost documentation site at https://docs.mattermost.com.
+
+Apply the style guide in the following situations:
+
+- when you create a new document
+- when you convert a document from Markdown to reStructuredText
```

Comment:
I think it's needed to be explicitly clear to prevent assumptions about what is meant.